pycbc/events/stat.py

```diff
@@ -567,6 +569,43 @@ def single(self, trigs):
                            dtype=numpy.float32), ndmin=1, copy=False)
 
 
+class CoincRateCalcStatistic(ExpFitStatistic):
```

Comment:
The name could use some work. Eg `ExpFitCoincRateStatistic` ?  Since it's based on expfit ..

Also, by making ExpFitStatistic the parent, the basic single-ifo SNR-like statistic that is being used is newsnr, not newsnr_sgveto.  I thought we wanted to do the latter .. so, make an init step that ensures this, or maybe make this a child of ExpFitSGCombinedSNR which does that already.  In that case the name should be eg `ExpFitSGCoincRateStatistic`.

Reply:
The `ExpFitStatistic` has the single function that we want, so have added the init step.

Did we want to use the `newsnr_sgveto_psdvar`?

Reply:
No. Newsnr_sgveto is the basic statistic we used in O2 and is the current basis for comparison & not decided to replace it yet.